packages/website/private/contents/post-008.mdx

```diff
@@ -0,0 +1,34 @@
+---
+slug: moving-to-our-github-organization
+title: Moving to Our GitHub Organization
+description: A major step toward making Waku a community-driven project
+author: daishi
+date: 2025/03/16
+---
+
+Waku started as my personal experimental project to explore React Server Components (RSC) and provide a playground for learning. Over the past two years, it has grown into something much bigger. It's a minimal React framework aiming to make development fun again.
+
+Today, we're excited to announce that Waku has officially moved to a GitHub organization: [https://github.com/wakujs](https://github.com/wakujs).
+
+## A Milestone Toward Openness
+
+This move marks an important milestone. Until now, Waku had been hosted under my personal GitHub account, but we wanted to explicitly signal that this is no longer just a personal project. This move makes it clear that it's a team effort.
+
+Technically, nothing changes in how we collaborate. However, this shift represents our commitment to making Waku an open and community-driven project. We're now more ready than ever to welcome contributors, receive feedback, and shape Waku together.
+
+## What's Next?
+
+We are currently finalizing **v0.22**, which will bring several improvements. Moving to a GitHub organization is just one step in the bigger picture of making Waku more usable and accessible to a wider audience.
+
+## How You Can Get Involved
+
+We'd love for you to be part of this journey. Here's how you can get started:
+
+- Read the [docs](https://waku.gg) and create a small app
+- Experiment with your favorite libraries
+- Try converting an existing app to Waku
+- Join discussions, share feedback, and contribute to the repo
```

Comment:
maybe a link to the discord server here?

Reply:
```suggestion
- Share your feedback in our [GitHub discussions](https://github.com/wakujs/waku/discussions) and in our [Discord server](https://discord.gg/MrQdmzd)
```